src/pages/Results-Basic.tsx

```diff
@@ -45,7 +46,7 @@ async function sendMessage(userInput: string, location: string): Promise<void> {
     }
 }
 
-
+// below is also chat GPT the one before the bar of dashes
```

Comment:
You don't need to specify where you're talking about if you mention the functionality of the statement / lines of code